integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/CoupleMBusDevice.feature

```diff
@@ -530,4 +530,4 @@ Feature: SmartMetering Installation - Couple M-Bus Device
     Examples:
       | GatewayDeviceIdentification | MbusPrimaryAddress | MbusIdentificationNumber | MbusManufacturerIdentification | MbusVersion | MbusDeviceTypeIdentification |
       | TEST1024000000001           |                  3 |                 12056731 |                            LGB |          66 |                            3 |
-      | TEST1024000000002           |                  0 |                        0 |                              0 |           0 |                            0 |
+#      | TEST1024000000002           |                  0 |                        0 |                              0 |           0 |                            0 |
```

Comment:
Waarom is deze test gedeactiveerd?

Reply:
Scherp, deze is inderdaad per abuis gedeactiveerd. Ik zet hem weer aan